project_template/inc/az/exampleshortname.h

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/** @file exampleshortname.h Azure example library */

```

Comment:
>exampleshortname [](start = 10, length = 16)

Consider having include guards so that the header file is not included twice

Reply:
this header doesn't include them because there should be nothing except includes of other code here. I'm not opposed to adding them (or `#pragma once`) but I don't feel it's necessary.